frontend/src/stats/types.ts

```diff
@@ -0,0 +1,18 @@
+export type ParsedFsrsCardStats = {
```

Comment:
Not sure if we can get this type def from the back-end codegen, but based on how things were being implement right now I assume not